Source/WTF/wtf/NativePromise.h

```diff
@@ -955,17 +956,28 @@ class NativePromise final : public NativePromiseBase, public ConvertibleToNative
         }
 
     private:
+        RefPtr<ThenCallbackType> protectedThenCallback()
+        {
+            return m_thenCallback;
+        }
+
+        Ref<NativePromise> protectedPromise()
```

Comment:
`m_promise` is never re-assigned, so we should mark the member variable `const` instead of making a `protectedPromise` function.